sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -150,7 +150,11 @@ abstract class HiveComparisonTest
        """.stripMargin
     })
 
-    super.afterAll()
+    try {
+      TestHive.reset()
```

Comment:
It doesn't hurt to throw in an extra `reset()` at the end of suite teardown here and might happen to fix some other isolation issues.